website/docs/dbt-cli/install/from-source.md

```diff
@@ -1,15 +1,18 @@
 ---
-title: "Install from source"
+title: "Install dbt from source"
 id: "from-source"
+description: "You can install dbt Core from its GitHub code source."
 ---
 
-dbt Core is open source software, as are almost all adapter plugins. As such, the codebases are freely available to download and build from source. To do this, you can clone the repositories from GitHub and install the local version using `pip`.
+dbt Core and almost all of its adapter plugins are open source software. As such, the codebases are freely available to download and build from source. You might install form source, if you want the latest code or want to install install dbt from a specific commit. This might be helpful when you are contributing changes, or if you want to debug a past change.
```

Comment:
```suggestion
dbt Core and almost all of its adapter plugins are open source software. As such, the codebases are freely available to download and build from source. You might install form source if you want the latest code or want to install dbt from a specific commit. This might be helpful when you are contributing changes, or if you want to debug a past change.
```